public/javascripts/SVLabel/src/SVLabel/keyboard/Keyboard.js

```diff
@@ -62,7 +62,13 @@ function Keyboard (svl, canvas, contextMenu, googleMap, ribbon, zoomControl) {
     this._documentKeyDown = function (e) {
         // The callback method that is triggered with a keyUp event.
         if (contextMenu.isOpen()) {
-            return;
+            if (e.keyCode == 16){
+                status.shiftDown = true;
+            }//only permits shift key value change when ctxt menu open
+            //kept separate from switch statement in elseif
+            if (e.keyCode == 187 || e.keyCode == 189){
```

Comment:
I added some comments. This was part of my attempt to disable +/- zoom, but it didn't work very well.